mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -7475,6 +7493,36 @@ resource "google_container_cluster" "primary" {
 `, name)
 }
 
+func testAccContainerCluster_withMonitoringConfigAdvancedDatapathMetricsEnabled(name string) string {
+       return fmt.Sprintf(`
+resource "google_container_cluster" "primary" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  monitoring_config {
+		advanced_datapath {
```

Comment:
nit: these multiline strings are generally two-spaced, while the Go portions of the file are tab-spaced. Mind cleaning up the spacing here & below? You can see this snippet mixes them.